lib/routes/miui/index.ts

```diff
@@ -1,10 +1,31 @@
-const got = require('@/utils/got');
-const queryString = require('query-string');
+import { Route } from '@/types';
+import got from '@/utils/got';
+import cache from '@/utils/cache';
+import queryString from 'query-string';
+
+export const route: Route = {
+    path: '/:device/:type?/:region?',
+    categories: ['program-update'],
+    example: '/miui/aries',
+    parameters: { device: 'the device `codename` eg. `aries` for Mi 2S', type: 'type', region: 'Region, default to `cn`' },
+    features: {
+        requireConfig: false,
+        requirePuppeteer: false,
+        antiCrawler: false,
+        supportBT: false,
+        supportPodcast: false,
+        supportScihub: false,
+    },
+    name: 'New firmware',
+    maintainers: ['Indexyz'],
+    description: "  | stable  | development |\n  | ------- | ----------- |\n  | release | dev         |\n\n  | region | region |\n  | ------ | ------ |\n  | China  | cn     |\n  | Global | global |",
```

Comment:
Please use [template literals](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Template_literals) which can include a visual linebreak instead of `\n`